app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(command: str) -> None:
+    file_names = command.split(" ")
```

Comment:
Consider using more descriptive variable names. For example, `file_names` could be `command_parts` to better reflect its purpose.